Logging/ARKLogDistributor.m

```diff
@@ -34,9 +34,13 @@ @interface ARKLogDistributor ()
 
 @property (atomic, assign) Class internalLogMessageClass;
 @property (atomic, weak) ARKLogStore *weakDefaultLogStore;
+@property (atomic, strong, readonly) NSRecursiveLock *defaultLogStorePropertyLock;
```

Comment:
non-blocking style nit: don't need the `strong` (#5 removes all the excessive modifiers)